keymaps/postcss-sorting.cson

```diff
@@ -1,8 +1,2 @@
-'.platform-darwin atom-text-editor':
-  'cmd-shift-s': 'postcss-sorting:run'
-
-'.platform-win32 atom-text-editor':
-  'ctrl-shift-s': 'postcss-sorting:run'
-
-'.platform-linux atom-text-editor':
+'atom-text-editor':
   'ctrl-shift-s': 'postcss-sorting:run'
```

Comment:
To be honest, I don't like this solution, because Mac users prefer `cmd` key-based shortcuts while Windows & Linux users use `ctrl` key. That's why I've separated this keymap depending on platform.